<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>449</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="plot_btn">
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press to plot chosen function&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="center_btn">
        <property name="text">
         <string>Show center</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear_btn">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="grid_btn">
        <property name="text">
         <string>Build std grid</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cones_checkBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>3D Cones</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="func_lbl">
        <property name="text">
         <string>Function </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="func_lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>cos(x)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="from_lbl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>From</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="from_spinBox">
        <property name="minimum">
         <number>-999999999</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>-10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="to_lbl">
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="to_spinBox">
        <property name="minimum">
         <number>-999999999</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dots_lbl">
        <property name="text">
         <string>Total dots</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="dots_spinBox">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="plot_wdgt">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
